yb-voyager/cmd/callhome_payload_builder.go

```diff
@@ -289,3 +293,70 @@ func getAnonymizedDDLs(sourceDBConf *srcdb.Source) []string {
 
 	return anonymizedDDLs
 }
+
+
+// ============================export schema callhome payload information============================
+
+func buildCallhomeSchemaOptimizationChanges() []callhome.SchemaOptimizationChange {
+	if schemaOptimizationReport == nil {
+		return nil
+	}
+
+	schemaOptimizationChanges := make([]callhome.SchemaOptimizationChange, 0)
+	if schemaOptimizationReport.RedundantIndexChange != nil {
+		objects := make([]string, 0)
+		for tbl, indexes := range schemaOptimizationReport.RedundantIndexChange.TableToRemovedIndexesMap {
```

Comment:
nit: a comment somewhere saying - anonymizing object name/identifiers before adding to callhome payload